src/app/examples/grid-angular.component.ts

```diff
@@ -11,13 +11,14 @@ import {
   Formatters,
   GridOption,
   OnEventArgs,
-} from './../modules/angular-slickgrid';
+} from 'angular-slickgrid';
```

Comment:
you cannot do that here, you need to put back previous code. This is the lib and we need to reference it, it is not installed as a `node_modules` but instead is used directly

Reply:
you're right